ui/src/App.vue

```diff
@@ -40,6 +40,14 @@ onMounted(async () => {
     if (translations[lang]) {
       mergeLocaleMessage(lang, translations[lang]);
 
+      nextTick(() => {
+        loaded.value = true;
+      });
+    } else {
+      const defaultTranslations = Reflect.ownKeys(translations)[0] as string;
+
+      mergeLocaleMessage(defaultTranslations, translations[defaultTranslations]);
+
       nextTick(() => {
         loaded.value = true;
       });
```

Comment:
I can't provide an answer based on the information provided because it is not clear what specific code you are referring to. Could you please specify the relevant code snippet?